src/v2/Apps/Consign/Routes/MarketingLanding/Components/InDemandNow/__tests__/ConsignInDemandNow.jest.tsx

```diff
@@ -31,16 +31,16 @@ describe("ConsignInDemandNow", () => {
     Component: (props: any) => {
       return (
         <MockBoot breakpoint={"lg" as Breakpoint}>
-          <tests.SoldRecentlyFragmentContainer
+          <tests.ConsignInDemandNowFragmentContainer
             targetSupply={props.targetSupply}
           />
         </MockBoot>
       )
     },
     query: graphql`
-      query SoldRecently_tests_Query {
+      query ConsignInDemandNow_tests_Query {
         targetSupply {
-          ...SoldRecently_targetSupply
+          ...ConsignInDemandNow_targetSupply
```

Comment:
Despite these being commented out; I'd get a warning for these whenever I opened force in VSCode because of how the Apollo extension tries to search for code

Reply:
Whats the warning?

Reply:
This file was copy pasted from another spec, so:

![](https://static.damonzucconi.com/_capture/fvbKToNk.png)

Reply:
Oh this fragment probably didn't matter; should probably just delete all this and write a pended spec

Reply:
ah, if commented out, probably a bug in the parser /shrug